src/SamplesApp/Benchmarks.Shared/Suite/Windows_UI_Xaml_Controls/DependencyPropertyBench/SimpleDPBenchmark.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using BenchmarkDotNet.Attributes;
+using Windows.Foundation;
+using Windows.UI.Xaml.Controls;
+
+namespace SamplesApp.Benchmarks.Suite.Windows_UI_Xaml_Controls.GridBench
+{
+	public class SimpleDPBenchmark
+	{
+		private Grid SUT;
+
+		[GlobalSetup]
+		public void Setup()
+		{
+			SUT = new Grid();
+		}
+
+		[Benchmark()]
+		public void DP_Write()
+		{
+			for (int i = 0; i < 100; i++)
+			{
+				SUT.Width = i;
+			}
+		}
+
+		[Benchmark()]
+		public void DP_Read()
+		{
+			for (int i = 0; i < 100; i++)
+			{
+				var r = SUT.Width;
```

Comment:
Are we sure the read doesn't get discarded in release ?

Reply:
It's considered a method invocation, so the compiler won't remove them because it can have a side effect.

Reply:
Good to know :)